python/manylinux1/scripts/install_cmake.sh

```diff
@@ -16,7 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
-/opt/python/cp37-cp37m/bin/pip install cmake ninja
+export CMAKE_VERSION=3.13.3 # 3.13.3 to avoid building 3.15.2 version from source
```

Comment:
Why do we need to use 3.13.3?

Reply:
pip tries to install 3.16.3 but there is no wheel for this version so it tries to build from source and fails. failed build - https://quay.io/repository/prudhvi/gandivacentos/build/57b9cb24-b6bc-4e27-a759-9e0aa910c9f5
pip was providing 3.13.3 version before 3.16.3

Reply:
Could you update the comment to describe this situation?

Reply:
done

Reply:
Thanks.